### PR TITLE
Add 1.1.0-msbuild

### PR DESCRIPTION
**Pull request by michimune**

### CODE REVIEW

File: 1.1.0-msbuild/init_container.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (nickwalkmsft) [inline 1.1.0-msbuild/init_container.sh]: .gitattributes for the line endings for this one?

Comment (michimune) [inline 1.1.0-msbuild/init_container.sh]: It's already there in my previous PR (Adding 1.0.0, not 1.1.0)

Comment (nickwalkmsft) [inline 1.1.0-msbuild/init_container.sh]: Ah you're right, forgot that was at the root and covered all matching filenames in the repo.

**Comment by michimune** [comment]
Thank you!

**Pull request merged**